js/js_test.go

```diff
@@ -558,3 +558,17 @@ func TestSurrogatePairs(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func instanceof(x interface{}, y *js.Object) bool {
+	return js.Global.Call("$instanceof", x, y).Bool()
+}
```

Comment:
I just realized that you added a new file for this. I'd prefer the `eval` method of line 15 for this. Also, could you maybe try to add a function to `dummys` that simply returns the parameter (thus externalizing the value) and then try `.Get("constructor") != uint8Array`?